issuer-web/public/config/config.json

```diff
@@ -10,7 +10,7 @@
   "inviteRequired": false,
   "deepLinkPrefix":"didcomm://launch",
   "authentication": {
-    "enabled": true,
+    "enabled": false,
```

Comment:
Intended change?

Reply:
No, that sneaked in. Not a concern though as it is the default file served with the app that is overwritten during deployment either way - used only for local development. Will revert to reduce noise.